systemtest/src/test/java/io/strimzi/systemtest/RollingUpdateST.java

```diff
@@ -229,15 +232,36 @@ void testKafkaAndZookeeperScaleUpScaleDown() throws Exception {
         LOGGER.info("Scaling up to {}", scaleTo);
         // Create snapshot of current cluster
         String kafkaStsName = kafkaStatefulSetName(CLUSTER_NAME);
-        KafkaResource.replaceKafkaResource(CLUSTER_NAME, k -> k.getSpec().getKafka().setReplicas(scaleTo));
+
+        KafkaResource.replaceKafkaResource(CLUSTER_NAME, kafka -> {
+            kafka.getSpec().getKafka().setReplicas(scaleTo);
+
+            JbodStorage jbodStorage =  new JbodStorageBuilder()
+                    .withVolumes(
```

Comment:
I don't get it. This adds JbodStorage with 7 volumes for each broker:
* i.e. 49 disks in total. So as a result, the counting later will not work because you expect only 7 / 3 volumes.
* You actually need to have similar storage configuration when you deploy the cluster, otherwise the CO will just ignore this change (which I guess is what happens because I guess you tried it and it is passing?)

I think this whole block can be easily removed.